superset-frontend/src/dashboard/components/SliceAdder.jsx

```diff
@@ -68,15 +76,20 @@ const KEYS_TO_SORT = {
   changed_on: t('recent'),
 };
 
-const DEFAULT_SORT_KEY = 'changed_on';
+export const DEFAULT_SORT_KEY = 'changed_on';
 
 const DEFAULT_CELL_HEIGHT = 128;
 
 const Controls = styled.div`
-  display: flex;
-  flex-direction: row;
-  padding: ${({ theme }) => theme.gridUnit * 3}px;
-  padding-top: ${({ theme }) => theme.gridUnit * 4}px;
+  ${({ theme }) => `
+    display: flex;
+    flex-direction: row;
+    padding:
+      ${theme.gridUnit * 4}px
+      ${theme.gridUnit * 3}px
+      ${theme.gridUnit * 4}px
```

Comment:
nit: it(bottom) was 3px in the previous style.

Reply:
@justinpark I intentionally changed it to adhere to Kasia's recommendation:

> let's bump the spacings between search and checkbox and thumbnails to 16px to give it more breathing space